src/frontend/komponenter/Felleskomponenter/KopierbartNullableFødselsnummer.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { formaterFødselsnummer } from '../../utils/formatter';
+import Clipboard from '@navikt/familie-clipboard';
+
+export const KopierbartNullableFødselsnummer: React.FC<{ fødselsnummer: string | undefined }> = ({
+    fødselsnummer,
+}) => {
+    return fødselsnummer ? (
+        <Clipboard>
```

Comment:
Her går det an å vurdere om vi vil ha `whitespace: nowrap`  for å alltid ha denne verdien på én linje.

Reply:
Enig :)